jujugui/static/gui/src/app/app.js

```diff
@@ -411,7 +411,35 @@ YUI.add('juju-gui', function(Y) {
         interactive: this.get('interactiveLogin'),
         serviceName: 'juju',
         dischargeStore: window.localStorage,
-        dischargeToken: dischargeToken
+        dischargeToken: dischargeToken,
+        visitMethod: response => {
```

Comment:
Any reason not to use a string of HTML, or react to render the notification? Perhaps in a follow up tidy up?